tests/project/test_definition_manager.py

```diff
@@ -21,7 +21,7 @@
 
 def mock_is_file_for(*known_files):
     def fake_is_file(self):
-        return str(self) in known_files
+        return str(self) in [str(Path(f)) for f in known_files]
```

Comment:
Why not `os.path.join(*f.split('/'))`? Does Path do the job on Windows? On posix it just assumes `\\` as valid character:
```
In [5]: Path('a\\b\\c').parent
Out[5]: PosixPath('.')
```

Reply:
`Path` is generic interface for building functions, the `pathlib` properly converts this to `WindowPath` on Win.